backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/CreateExperimentRunHandler.java

```diff
@@ -386,4 +384,8 @@ private String buildRoleBindingName(
             },
             executor);
   }
+
+  public InternalFuture<Void> createClonedExperimentRun(ExperimentRun experimentRun) {
+    return insertExperimentRun(experimentRun);
+  }
```

Comment:
https://github.com/VertaAI/docs/blob/main/engineering-practices/api-implementation/performance.md#avoid-layers-of-abstraction

Reply:
Improve code to resolved abstraction for clone ER